src/Canvas.elm

```diff
@@ -7,6 +7,24 @@ import Types.Kurve exposing (Kurve)
 import World exposing (DrawingPosition)
 
 
+type CanvasAction
+    = DrawSomething WhatToDraw
+    | ClearEverything
```

Comment:
You can also consider adding `| NoAction` to this type, instead of having `Maybe CanvasAction`, to make it feel more like `Cmd.none` and avoiding `Just`. I’ve seen that in other projects. But it doesn’t really matter – it’s about taste.

Reply:
I went down your suggested path, and then went full Effect pattern, with the no-op variant named `DoNothing`.